app/services/food_service.rb

```diff
@@ -0,0 +1,17 @@
+class FoodService
+
+  def self.conn
+    Faraday.new(url: 'https://api.nal.usda.gov') do |f|
+      f.params['api_key'] = ENV['food_key']
+    end
+  end
+
+  def self.json_response(response)
```

Comment:
In here I am wondering if you could pass additional params like pageSize to limit 10